Pipelines/core-pipeline.yml

```diff
@@ -43,7 +45,7 @@ stages:
     - task: DotNetCoreCLI@2
       displayName: Dotnet Test
       inputs:
-        command: 'test'
+        command: 'test --filter ClassName=Microsoft.CST.OpenSource.Tests.DetectCryptographyTests.cs|ClassName=Microsoft.CST.OpenSource.Tests.DownloadTests.cs|ClassName=Microsoft.CST.OpenSource.Tests.FindSourceTests.cs|ClassName=Microsoft.CST.OpenSource.Tests.VersionTests.cs'
```

Comment:
Is there a better way of doing this that doesn't involve hardcoding the test names here?

Reply:
Fixed.